package.json

```diff
@@ -192,6 +192,7 @@
     "react-router-dom": "^4.2.2",
     "react-select": "^1.0.0",
     "react-simple-file-input": "^2.0.0",
+    "react-table": "7.1.0",
```

Comment:
We should not use a specific version at least there is some reason for that

Reply:
Not a reason in particular, there i updated it to be always the latest version